WordPress/WordPressUITests/Screens/LoginPasswordScreen.swift

```diff
@@ -12,20 +12,29 @@ class LoginPasswordScreen: BaseScreen {
     }
 
     func proceedWith(password: String) -> LoginEpilogueScreen {
-        tryProceed(password: password)
+        _ = tryProceed(password: password)
+
+        let alertModal = XCUIApplication().alerts["“WordPress” Would Like to Send You Notifications"]
+        if alertModal.waitForExistence(timeout: 3) {
+            alertModal.buttons["Don’t Allow"].tap()
+        }
+
         return LoginEpilogueScreen.init()
     }
 
     func tryProceed(password: String) -> LoginPasswordScreen {
         passwordTextField.tap()
         passwordTextField.typeText(password)
         loginButton.tap()
-        waitFor(predicate: "isEnabled == true", element: loginButton)
+        if loginButton.exists && !loginButton.isHittable {
+            waitFor(element: loginButton, predicate: "isEnabled == true")
+        }
         return self
     }
 
     func verifyLoginError() -> LoginPasswordScreen {
-        XCTAssert(XCUIApplication().staticTexts["pswdErrorLabel"].exists)
+        sleep(1) // Give some time for error text to appear
```

Comment:
Will not be better to wait for the label to appear with a timeout? Something like this: 

```
let app = XCUIApplication()
app.launch()

let label = app.staticTexts["Hello, world!"]
let exists = NSPredicate(format: "exists == 1")

expectationForPredicate(exists, evaluatedWithObject: label, handler: nil)
waitForExpectationsWithTimeout(5, handler: nil)
```